src/Compilers/CSharp/Portable/Symbols/NullableAnnotation.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// The nullable annotations that can apply in source.
+    /// </summary>
+    internal enum NullableAnnotation : byte
+    {
+        /// <summary>
+        /// No information. Think oblivious.
+        /// </summary>
+        Unknown,
```

Comment:
Why not `Oblivious` here? All the helpers that use the value use this term.

Reply:
For a while we thought we should use "unknown" instead of "oblivious". 
But now, Mads' spec bolsters the term "oblivious", so we may need to revisit.

We have a tracking issue already https://github.com/dotnet/roslyn/issues/29632 (this affects docs and public API)